test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs

```diff
@@ -101,6 +107,9 @@ public void ChangingDropdownEngineSavesCode()
 
             //assert model code is updated.
             Assert.AreEqual("OUT = 100", (nodeModel as PythonNode).Script);
+            DispatcherUtil.DoEvents();
+            Run();
```

Comment:
Do you need to call run manually? I'm confused now

Reply:
This graph is in manual mode.
https://github.com/DynamoDS/Dynamo/blob/master/test/core/python/python.dyn#L55

Reply:
Oh. Should we use that uses auto then? With manual mode you wouldn't be able to reproduce this issue in the first place.

Reply:
sure, I'll find a python graph in auto.

Reply:
Ok. Then feel free to ignore my previous comment or go all the way and use auto mode. Sorry about the misunderstanding.

Reply:
ok, instead of finding a python graph with auto, I just set the runtype to auto right after opening it, and I removed the manual run call.